lib/utils/astutils.ml

```diff
@@ -44,6 +44,11 @@ let is_exp_const : exp -> bool = fun e ->
   | Pexp_constant _ -> true
   | _ -> false
 
+let is_exp_bool : exp -> bool = fun e ->
```

Comment:
```suggestion
let is_bool_lit : exp -> bool = fun e ->
```
Just for naming consistency with the other functions in `equality.ml`, since what we're really checking for here is that the given expression is a boolean literal e.g. true or false